airflow-core/src/airflow/jobs/scheduler_job_runner.py

```diff
@@ -1315,6 +1315,11 @@ def _run_scheduler_loop(self) -> None:
                 self._emit_running_ti_metrics,
             )
 
+            timers.call_regular_interval(
+                conf.getfloat("scheduler", "dagrun_metrics_interval", fallback=30.0),
```

Comment:
Does this config exist? I don't see an update to config_templates/config.yml here?